ocaml/forkexecd/lib/forkhelpers.ml

```diff
@@ -40,20 +40,40 @@ let with_tracing ~tracing ~name f = Tracing.with_tracing ~parent:tracing ~name f
 
 let finally = Xapi_stdext_pervasives.Pervasiveext.finally
 
-type pidty = Unix.file_descr * int
+let use_daemon = ref false
```

Comment:
Currently changing the source and recompiling. I have a branch that uses it, so I know that it works. Indeed an external setting would be useful. Suggestions (environment, configuration file, program options, links to Xapi code for it) ?

Reply:
Because this is a library loaded from different processes, using diverse methods; checking for a file to be present could be a convenient way to go. Regarding the location, something like `/etc/xensource/forkexec-uses-daemon`?

Reply:
then, what about a simple
```ocaml
let use_daemon = Sys.file_exists "/etc/xensource/forkexec-uses-daemon"
```

Reply:
That works for me